src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -208,8 +219,12 @@ export const KebabCatalogTileBadge: React.FC<KebabCatalogTileBadgeProps> = ({ te
       <DropdownItem key={'delete'} onClick={handleTemplateDelete}>
         {t('DELETE', { ns: 'common' })}
       </DropdownItem>,
+      <DropdownItem key={'clear'} onClick={handleClearCards}>
+        {t('CLEAR_ALL_CARDS', { ns: 'common' })}
```

Comment:
Template should not be modified. Only layout is allowed to be cleared. Could u remove changes to this file?